### PR TITLE
Ocrvs 4189 - Clicking "Create" in the "Create client" -modal will call a mutation in the GraphQL gateway, and populate "Client ID", "Client Secret" and "Sha secret"

### PR DESCRIPTION
**Pull request by CornelMaze**
- Created mutation to populate "Client ID", "Client Secret" and "Sha secret".

### CODE REVIEW

File: packages/components/src/icons/Copy.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/*
```
Comment (jpye-finch) [inline packages/components/src/icons/Copy.stories.tsx:1]: @CornelMaze we have a component called <Icon />

The copy icon used is feather icon called copy. So this file and story is not required

File: packages/client/src/views/SysAdmin/Config/Integrations/Integrations.tsx [hunk 2]
```diff
@@ -35,9 +49,94 @@ export const statuses = {
   DEACTIVATED: 'deactivated'
 }
 
-function Integrations() {
+const PaddedAlert = styled(Alert)`
```
Comment (jpye-finch) [inline packages/client/src/views/SysAdmin/Config/Integrations/Integrations.tsx:52]: You can make use of < Stack /> go remove alot of these custom components

Comment (CornelMaze) [reply packages/client/src/views/SysAdmin/Config/Integrations/Integrations.tsx:52]: Ok. Would address this as well.

**Comment by Zangetsu101** [comment]
@CornelMaze I see a lot of commits not related to your Ticket in this PR

**Comment by CornelMaze** [comment]
> @CornelMaze I see a lot of commits not related to your Ticket in this PR

This is based on favour's pr.

**Comment by naftis** [comment]
https://github.com/opencrvs/opencrvs-core/pull/4181 Merge this one first
Then pull the integration-configuration branch to this one.
Fix potential conflicts.
Merge.

**Pull request merged**